### ISSUE TITLE
Lumen - Call to undefined function Spatie\Permission\resolve() in v3.4.0

### ISSUE THREAD

**Issue by abishekrsrikaanth** [spatie/laravel-permission#1311]
I get the following error after getting the latest update. Throws this update when running `php artisan`

```
In PermissionRegistrar.php line 88:
                                                          
  Call to undefined function Spatie\Permission\resolve() 
```

PR https://github.com/spatie/laravel-permission/pull/1310 at https://github.com/spatie/laravel-permission/commit/4595666e79a44aaec690cd4149d8a270b506cf3a#diff-d9395f15dac405a38bdf8c96762a9a6eR88

**Comment by drbyte** [comment]
Is this in a Laravel app? What version?

It's working fine in a Laravel 6.9.0 app that I just applied the update to. Running `php artisan` gives no errors.

`resolve()` is a function provided by Laravel core, via `/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php`
(It's essentially an alias to `app()`, also from that same file.)

Why is your app not finding standard Laravel resources?

**Comment by abishekrsrikaanth** [comment]
I am on Lumen v6.2.0 which is the latest.

**Comment by drbyte** [comment]
Right.

Fixed by https://github.com/spatie/laravel-permission/commit/cbdca3c1be8461fa6f411dca68638c452b1062ba
Released as [3.4.1](https://github.com/spatie/laravel-permission/releases/tag/3.4.1)
